vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/exporter/ZephyrExporter.java

```diff
@@ -40,18 +41,21 @@
     private final JiraFacade jiraFacade;
     private IZephyrFacade zephyrFacade;
     private TestCaseParser testCaseParser;
+    private ZephyrExporterProperties zephyrExporterProperties;
 
-    public ZephyrExporter(JiraFacade jiraFacade, ZephyrFacade zephyrFacade, TestCaseParser testCaseParser)
-            throws IOException
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public ZephyrExporter(JiraFacade jiraFacade, ZephyrFacade zephyrFacade, TestCaseParser testCaseParser,
+            ZephyrExporterProperties zephyrExporterProperties) throws IOException
     {
         this.jiraFacade = jiraFacade;
         this.zephyrFacade = zephyrFacade;
         this.testCaseParser = testCaseParser;
+        this.zephyrExporterProperties = zephyrExporterProperties;
     }
 
     public void exportResults() throws IOException
     {
-        ObjectMapper objectMapper = new ObjectMapper();
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         objectMapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
         objectMapper.registerModule(new SimpleModule().addDeserializer(TestCase.class, new TestCaseDeserializer()));
```

Comment:
move configuration of `ObjectMapper` to constructor to avoid possible duplicate re-configuration